Cargo.toml

```diff
@@ -44,11 +44,11 @@ once_cell = "1.17.1"
 sha2 = "0.10"
 serde = "1"
 serde_json = "1"
-penumbra-component = { git = "https://github.com/penumbra-zone/penumbra.git", rev = "ab1f82970af76091b962509831ec0f53cfeb85f4" }
-penumbra-ibc = { git = "https://github.com/penumbra-zone/penumbra.git", rev = "ab1f82970af76091b962509831ec0f53cfeb85f4", default-features = false }
-penumbra-proto = { git = "https://github.com/penumbra-zone/penumbra.git", rev = "ab1f82970af76091b962509831ec0f53cfeb85f4" }
-penumbra-storage = { git = "https://github.com/penumbra-zone/penumbra.git", rev = "ab1f82970af76091b962509831ec0f53cfeb85f4" }
-penumbra-tower-trace = { git = "https://github.com/penumbra-zone/penumbra.git", rev = "ab1f82970af76091b962509831ec0f53cfeb85f4" }
+cnidarium = { git = "https://github.com/penumbra-zone/penumbra.git", tag = "v0.64.1" }
```

Comment:
please mention the rename in the PR description.

Also, should we have a standing issue to migrate to crates.io? Looks like they are prepping a crate?